mailpoet/lib/Migrations/Db/Migration_20230831_143755_Db.php

```diff
@@ -52,6 +52,9 @@ public function run(): void {
 
     // add unique index, remove no longer needed index
     if (!$this->indexExists($logsTable, 'automation_run_id_step_id')) {
+      $this->connection->executeStatement(
+        "DELETE t1 FROM $logsTable as t1, $logsTable as t2 WHERE t1.id < t2.id AND CONCAT(t1.automation_run_id,t1.step_id) = CONCAT(t2.automation_run_id,t2.step_id); "
+      );
```

Comment:
@websupporter @JanJakes pointed out to me that concat can be problematic. E.g., run id1 and step id1 are 1, 11, and run id2 and step id2 are 11, 1.  
It can be replaced by multiple conditions
```sql
DELETE t1 FROM $logsTable as t1, $logsTable as t2 WHERE t1.id < t2.id AND t1.automation_run_id = t2.automation_run_id AND t1.step_id = t2.step_id
```

Reply:
:facepalm: when you forget to do `git add` again after @JanJakes points out to me that concat can be problematic